meet-data/ipf/ipf-standardize-csv

```diff
@@ -124,41 +153,106 @@ def unreverse_names(csv):
 
         row[nameidx] = name
 
+def add_events(csv,events):
+    eventidx = csv.index('Event')
+    for row in csv.rows:
+        row[eventidx]=events
 
-def main(filename):
-    csv = Csv()
-    with open(filename, 'r') as fd:
-        csv.rows = [x.strip().split(',') for x in fd.readlines()]
+def add_sex(csv,sex):
+    sexidx = csv.index('Sex')
+    for row in csv.rows:
+        row[sexidx]=sex
+
+
+#Finds the sex of the lifters in a meet from the weight classes
+def meet_sex(csv):
+    mens_classes = ['-53','-59','-66','-74','-83','-93','-105','-120','120+']
+    womens_classes = ['-43','-47','-57','-63','-72','-84','84+'] #Don't have -52 as this overlaps with an old class
```

Comment:
Older meets use the old system, where men and women have the same weightclasses. (The "Traditional" classes.)

Reply:
The old_mens_only and old_womens_only arrays are there for meets which use the old weight-classes. The old weight classes are still slightly different in that there are women's 44,48,90+ classes and no women's classes for 100,110,125,125+ - so lifter sex should still be able to be determined from the weight classes.

 Since the script doesn't know which era weight-classes it's dealing with I removed 52kg from the new women's weight classes - otherwise a -52kg men's class may have been flagged as a women's class.